packages/ui/src/modal/bid/TokenStatsHeader.tsx

```diff
@@ -20,7 +20,9 @@ type Props = {
 
 const TokenStatsHeader: FC<Props> = ({ token, collection }) => {
   const img = optimizeImage(
```

Comment:
Optimize image was kind of meant to do this on the client but we should remove it completely from the code as it's not really necessary.

Reply:
Ok sweet, will remove it 👍

Reply:
On second thought, I kept it in there because the collection image doesn't have the dynamic url support